### PR TITLE
RK-16293 - RK-16294 - labels annotations docs

### PR DESCRIPTION
**Pull request by gosharo**
- Labels annoations and docs
- pvc fix

### CODE REVIEW

**Comment by gosharo** [comment]
RK-16294

**Comment by gosharo** [comment]
RK-16293

File: charts/rookout-hybrid/README.md [hunk 1]
```diff
@@ -23,12 +23,37 @@ helm repo update
...
+    --create-namespace rookout
+```
+
+### Datastore access
+
+This installation will not expose your datastore to your web-browser (Rookout client), ingress or other method you are using should expose the datastore service to your web-browser.
+
+<img src="https://docs.rookout.com/assets/images/datastore_diagram-628f79ca44d7af9c355c6f0f7d821712.png" width="900">
+
+you can expose datastore using ingress or chaning the service type of it to LoadBalancer or NodePort as preffered. for reference checkout [Values.yaml](https://github.com/Rookout/helm-charts/tree/master/charts/rookout-hybrid/values.yaml)
```
Comment (naomirookout) [inline charts/rookout-hybrid/README.md:40]: typo - change "chaning" to "chaining"

Comment (naomirookout) [reply charts/rookout-hybrid/README.md:40]: grammar - change "checkout" to "check out"

File: charts/rookout-hybrid/README.md [hunk 2]
```diff
@@ -23,12 +23,37 @@ helm repo update
...
 ```commandline
-helm upgrade --install rookout rookout/rookout-hybrid -f ingerss_example.yaml
+helm upgrade --install rookout rookout/rookout-hybrid \
+    --set rookout.token="YOU_TOKEN_HERE" \
+    --create-namespace rookout
+```
+
+### Datastore access
+
+This installation will not expose your datastore to your web-browser (Rookout client), ingress or other method you are using should expose the datastore service to your web-browser.
```
Comment (naomirookout) [inline charts/rookout-hybrid/README.md:36]: Change to "This installation will not expose your datastore to your web-browser (the Rookout client), unless granted explicitly."

File: charts/rookout-hybrid/README.md [hunk 3]
```diff
@@ -23,12 +23,37 @@ helm repo update
...
+
+<img src="https://docs.rookout.com/assets/images/datastore_diagram-628f79ca44d7af9c355c6f0f7d821712.png" width="900">
+
+you can expose datastore using ingress or chaning the service type of it to LoadBalancer or NodePort as preffered. for reference checkout [Values.yaml](https://github.com/Rookout/helm-charts/tree/master/charts/rookout-hybrid/values.yaml)
+
+Here is naive example using ingress, nginx and let's encrypt issuer.
```
Comment (naomirookout) [inline charts/rookout-hybrid/README.md:42]: Grammar - "Here is" --> "Here is a"

File: charts/rookout-hybrid/README.md [hunk 4]
```diff
@@ -23,12 +23,37 @@ helm repo update
...
+[nginx_lets_encrypt.yaml](https://github.com/Rookout/helm-charts/tree/master/charts/rookout-hybrid/example/nginx_lets_encrypt.yaml)
+
+### Rooks configuration
+
+If the rooks (agents) in same cluster as Rookout's controller they can communicate using internal k8s DNS server, the address will be `CONTROLLER_SERVICE_NAME.NAMESPACE.svc.cluster.local` the default domain with the above command would be `controller-rookout-rookout-hybrid.rookout.svc.cluster.local`. If the rooks not in same cluster, expose the controller's service as you did for the datastore.
```
Comment (naomirookout) [inline charts/rookout-hybrid/README.md:47]: Change to
"If the agents are in same cluster as Rookout's controller, they can communicate using the internal k8s DNS server. The address will be `CONTROLLER_SERVICE_NAME.NAMESPACE.svc.cluster.local` and the default domain will be `controller-rookout-rookout-hybrid.rookout.svc.cluster.local`. If the agents are not in same cluster, you must expose the controller's service as you did for the datastore."

File: charts/rookout-hybrid/README.md [hunk 5]
```diff
@@ -23,12 +23,37 @@ helm repo update
...
+
+To configure the rooks (agents) after integration of the rook, please pass the following enviorment variables:
```
Comment (naomirookout) [inline charts/rookout-hybrid/README.md:49]: Change to "After integrating the agents, configure them by passing the following environment variables:"

File: charts/rookout-hybrid/README.md [hunk 6]
```diff
@@ -23,12 +23,37 @@ helm repo update
...
+```
+- name: ROOKOUT_CONTROLLER_HOST
+  value: 'ws://controller-rookout-rookout-hybrid.rookout.svc.cluster.local'
+- name: ROOKOUT_CONTROLLER_PORT
+  value: '80'
+- name: ROOKOUT_TOKEN
+  value: 'YOUR_TOKEN'
```
Comment (naomirookout) [inline charts/rookout-hybrid/README.md:56]: Let's be consistent with our empty state value for the Rookout token - above this is YOU_TOKEN_HERE.

**Review by naomirookout — DISMISSED** [review]

**Review by alexeygutkin — APPROVED** [review]

**Pull request merged**